templates/InstallChocolatey.ps1.erb

```diff
@@ -100,10 +100,14 @@ if ($unzipMethod -eq '7zip') {
   Write-Output "Extracting $file to $tempDir..."
   Start-Process "$7zaExe" -ArgumentList "x -o`"$tempDir`" -y `"$file`"" -Wait -NoNewWindow
 } else {
-  $shellApplication = new-object -com shell.application
-  $zipPackage = $shellApplication.NameSpace($file)
-  $destinationFolder = $shellApplication.NameSpace($tempDir)
-  $destinationFolder.CopyHere($zipPackage.Items(),0x10)
+  if ($PSVersionTable.PSversion.major -lt 5) {
+    $shellApplication = new-object -com shell.application

```

Comment:
No dice - needs at least 4.5 for ZipFile - https://msdn.microsoft.com/en-us/library/system.io.compression.zipfile(v=vs.110).aspx